bsconfig.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "bs-date-fns",
   "bsc-flags": ["-bs-super-errors"],
-  "reason": {"react-jsx": 2},
+  "reason": {"react-jsx": 3},
```

Comment:
Hi! Can we remove this line altogether? Doesn't look like React/JSX is used anywhere in this library, but it does mess with bs-platform 8.2.0.

Reply:
Sure